deployer/prepare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
```

Comment:
I was hoping you would have a better/cleaner idea for how to do this? :sweat_smile:

Reply:
Else, I'll just comment these properly

Reply:
Extra comments added

Reply:
> I was hoping you would have a better/cleaner idea for how to do this? 😅

Not really I'm afraid! The only things I can think of are:
- Just add `RUN cargo fetch` in the `shuttle-common` image building step. It does mean the base layer gets a bit bigger, but should not impact things too much since the size difference can't be immense. Or:
- have this be an "init" step of deployer, triggered from there

Reply:
For interest sake, this increases the deployer container size from 2Gb to 3Gb and the compile speedup is quite huge on my system